core/pom.xml

```diff
@@ -215,6 +215,11 @@
             <groupId>io.swagger</groupId>
             <artifactId>swagger-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>confluent-metrics</artifactId>
+            <version>${ce.kafka.version}</version>
```

Comment:
since this is a public repo, I don't think we can depend on proprietary jars

Reply:
Removed, thank you !